Tests/BindTests/RelayTests.swift

```diff
@@ -0,0 +1,31 @@
+import XCTest
+@testable import Bind
+
+final class RelayTests: XCTestCase {
+    func testRelayNotFired() {
+        let expect = expectation(description: "not called")
+        expect.isInverted = true
+
+        let relay = Relay()
+
+        relay.bind { _ in
+            expect.fulfill()
+        }
+
+        waitForExpectations(timeout: 1, handler: nil)
```

Comment:
This timeout possibly a bit long? Will always cause the tests to take an extra second, could probably be 0.1